Improve romanian language

Danny3 opened the pull request:
Hi guys,

I made some chages to improve the romanian language support
There are 2 important changes:

Added romanian language to the  TMDb add-on settings page, this fixes #15823
Added romanian keyboard layout

Imust tell you that I'm new to git and github, so please check if everything is ok

Thank you

Commit:
Update romanian langinfo.xml

Use full name for Moldova
Add back symbols for AM and PM
add 2 articles to ignore when sorting

Commit:
Update romanian language description for themoviedb add-on

Update romanian language description for themoviedb add-on

Commit:
Add romanian language to themoviedb add-on settings page

Add romanian language to themoviedb add-on settings page

Commit:
Add romanian keyboard layout

Add romanian keyboard layout to the keyboard layouts list
Since the second row cannot display more than 12 letters, one letter is
moved on the next row.

MartijnKaijser commented:
we don't do any changes any more for Helix

Montellese commented:
The romanian keyboard layout could certainly be ported to master (even though the file structure has changed a bit). For the language updates you need to register on Transifex and request to be added to the romanian language team.

Danny3 commented:
I am part of the romanian language team, and the articles that I added to ignore when sorting I already added to the description string of ignore articles seting on Transifex.

For the keyboard I will wait some time and resubmit with the new structure.
Thank you
